ros2_build.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -xe
 
-# add keys and sources
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
```

Comment:
This is also the old key that should no longer be used anyways (https://discourse.ros.org/t/new-gpg-keys-deployed-for-packages-ros-org/9454).